conclave-common/src/main/kotlin/com/r3/conclave/common/MockCallInterfaceConnector.kt

```diff
@@ -0,0 +1,58 @@
+package com.r3.conclave.common
```

Comment:
This is a public API package, and this class is internal. Move this to the internal package.